packages/backend/src/core/liveness/LivenessIndexer.ts

```diff
@@ -40,30 +45,35 @@ export class LivenessIndexer extends ChildIndexer {
   }
 
   override async update(from: number, to: number): Promise<number> {
-    const configurations = await this.livenessConfigurationRepository.getAll()
-
-    const { data, adjustedTo, usedConfigurationsIds } =
-      await this.livenessClient.getLivenessData(
-        this.projects,
-        configurations,
-        new UnixTime(from),
-        new UnixTime(to),
-      )
+    const { transfersConfig, functionCallsConfig, adjustedTo } =
+      await this.getConfiguration(from, to)
+
+    const data = await this.livenessClient.getLivenessData(
+      transfersConfig,
+      functionCallsConfig,
+      new UnixTime(from),
+      adjustedTo,
+    )
 
     await this.livenessRepository.runInTransaction(async (trx) => {
       await this.livenessRepository.addMany(data, trx)
 
       await Promise.all(
-        usedConfigurationsIds.map((id) =>
+        [...transfersConfig, ...functionCallsConfig].map((c) =>
           this.livenessConfigurationRepository.setLastSyncedTimestamp(
-            id,
+            c.livenessConfigurationId,
             adjustedTo,
             trx,
           ),
         ),
       )
     })
 
+    const usedConfigurationsIds = [
```

Comment:
there is no need to map, please return transfer configs length and function configs length, so we will more granularily see what happened